src/components/Tank/Tank.model.ts

```diff
@@ -0,0 +1,14 @@
+export declare type TankPosition = 'up' | 'down' | 'left' | 'right';
+export declare type TankColor = 'yellow' | 'white' | 'green' | 'red';
+export declare type TankType = '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7';
+
+export interface TankProps {
+  color?: TankColor;
+  position?: TankPosition;
+  type?: TankType;
```

Comment:
- [ ] Why did you make all the properties optional? I don't think they should be.